src/opm/parser/eclipse/share/keywords/000_Eclipse100/D/DIFFC

```diff
@@ -10,11 +10,15 @@
   "items": [
     {
       "name": "OIL_MOL_WEIGHT",
-      "value_type": "DOUBLE"
+      "value_type": "DOUBLE",
+      "default": -1.0,
```

Comment:
I am *very* sceptical to use of magic numbers like `-1` - are your *really really* sure that is needed?

Reply:
I want the OIL_MOL_WEIGHT to be optional. 

What is the preferred way to handle
DIFFC 
 2* 1-e5 1e-6 

if DIFFC item 1 is defaulted do this otherwise do that kind of logic. Can I use the has_default logic?

Reply:
Using the default applied logic is not trivial using a FlatTable. I will instead remove the option of using default molar weights. It probably complicates life more than it simplifies it. I adjust the PR accordingly.